src/ol_infrastructure/applications/airbyte/Pulumi.airbyte.applications.QA.yaml

```diff
@@ -2,14 +2,16 @@
 secretsprovider: awskms://alias/infrastructure-secrets-qa
 encryptedkey: AQICAHi7xhTkB8tf1ObyPMxDhODJja4Mn4jyIo32zZVlOiZPFgFqyheTEwQhQwt19Cm5Z2wyAAAAfjB8BgkqhkiG9w0BBwagbzBtAgEAMGgGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQMUHRADb5VO/Osd+QMAgEQgDthoUItjofWZZROiCvTVCFgPdVM1f3tWDvxoezDTWseixH0X/RXXDnIDnOXDiC9dHp/ZRoXORovMifXGg==
 config:
+  airbyte:auth_domain: airbyte-qa-auth.odl.mit.edu
```

Comment:
Maybe `auth.airbyte-qa.odl.mit.edu`? Doesn't actually matter, just a stylistic nit

Reply:
Will that work with our wildcard?

Reply:
Good point, I had read it as `airbyte-qa.auth` (note the dot, not dash). Go ahead and keep as is